epitator/date_annotator.py

```diff
@@ -78,13 +158,14 @@ def date_to_datetime_range(text, relative_base=doc.date, prefer_dates_from='past
                 date_spans.append(ne_span)
         # Regex for formatted dates
         regex = re.compile(
+            # date MonthName yyyy
```

Comment:
Are we sure these are the only date formats we want to match?  Looking at the `date MonthName yyyy` format it occurs to me that when I say my birthday I usually use the format `MonthName date yyyy` - should we be matching for this as well?

Also - the comments say these match on dates formatted with dashes (`-`) but they will also match date formatted with forward slashes(`/`) but not back slashes (`\`).  I'm not sure if it matters, but these will match across line breaks as well.

In summary, these will match:
`11/11/1111`
```
11-
11-
1111
```

and these won't:
`11\11\1111`

Will this be ok in our situation?